.github/workflows/build.yml

```
@@ -823,6 +823,7 @@ jobs:
           EOF
           )
           if [ "$TARGET_LIBRARY" = voicevox_onnxruntime ]; then
+            release_notes+=$'\n'
             release_notes+=$'\n'
```

こうしておく手もあるかも（これ構文的にあってるのかな･･･）
```suggestion
            release_notes+=$'\n\n' # NOTE: 2行なのはGFMで見た目が崩れるのを防ぐため
```

全然強い意見じゃないです！　再発しそうかも、くらい！

---

ですね。そもそもこれNOEOL状態にEOL追加 + さらに空行という形なので、その方がよさそう。